### ISSUE TITLE
[PT] Update lesson: GenStage

### ISSUE THREAD

**Issue by thiamsantos** [elixirschool/elixirschool#2397]
## Motivation

The portuguese translation for the lesson [GenStage](https://elixirschool.com/en/lessons/advanced/gen-stage/) needs to be updated.
We need to update the version of the lesson from `1.1.0` to `1.1.1`.

## Proposed Solution

Translate the updates of the lesson.

You can run `git diff` to see all differences between the versions of the lesson:

```sh
git diff 09b28344300e5c395322e972ea085247cdcefbd4 HEAD en/lessons/advanced/gen-stage.md
```

## Tips

- Check the [contributing guidelines](https://github.com/elixirschool/elixirschool/blob/master/CONTRIBUTING.md) before starting.
- Feel free to open a [draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/#tag-your-work-in-progress) as soon as you start working on the translation. In order to prevent we prevent having multiple people working on the translation and also allow the reviewers to give early feedback.
- In case of any doubt feel free to comment in this issue.

**Comment by herminiotorres** [comment]
I take this update lesson and I just sent a pull-request #2406

**Comment by thiamsantos** [comment]
Closed by #2406 

Thanks @herminiotorres
